core/cap-0035.md

```diff
@@ -483,6 +528,24 @@ the issuer of the `asset`.
 - `CLAWBACK_CLAIMABLE_BALANCE_NOT_CLAWBACK_ENABLED` if the
 `CLAWBACK_ENABLED_FLAG` is not set.
 
+#### Set Trustline Flags Operation
+
+This proposal introduces the `SetTrustlineFlagsOp` operation. The operation works
+like the `AllowTrustOp`, except it uses set and clear parameters to set/clear specific
+trustline flags. This will allow an issuer to clear the `CLAWBACK_ENABLED_FLAG`.
+
+Possible return values for the `SetTrustlineFlagsOp` are:
+- `SET_TRUSTLINE_FLAGS_SUCCESS` if the operation is successful.

```

Comment:
❓ Do we need result codes similar to `ALLOW_TRUST_TRUST_NOT_REQUIRED`? Or no because of [CAP-29](https://stellar.org/protocol/cap-29)?

What about `ALLOW_TRUST_SELF_NOT_ALLOWED`?

Reply:
Yeah `ALLOW_TRUST_TRUST_NOT_REQUIRED` won't be needed for the reasons specified in CAP-29. `SET_TRUSTLINE_FLAGS_MALFORMED` can be returned instead of a different result code since this check can be done at validation time. We return `CLAWBACK_MALFORMED` if you try to clawback from self (I missed this case as one of the conditions in the CAP for both malformed results, so I'll make sure to update them).